lightning/src/ln/channelmanager.rs

```diff
@@ -8714,7 +8738,7 @@ where
 	/// [`MAX_SHORT_LIVED_RELATIVE_EXPIRY`].
 	fn create_blinded_path_using_absolute_expiry(
```

Comment:
s/path/paths

Likewise for the two methods below. Please also make "path" plural in the docs.

Reply:
Done! [4461115](https://github.com/lightningdevkit/rust-lightning/commit/4461115e88f7fb3e0a366a24aba8a32bc6c7b408)
Thanks!